lib/viewmodels/map/map_view_model.dart

```diff
@@ -59,28 +59,30 @@ class MapViewModel extends AutoDisposeAsyncNotifier<MapDetails> {
     state = await AsyncValue.guard(() => build());
   }
 
-  static const _initialZoomLevel = 17.0;
-
-  double get initialZoom => _initialZoomLevel;
+  static const initialZoomLevel = 17.0;
```

Comment:
Thanks for making it public :+1: 
I wonder if we can add a little line of doc too :)

Reply:
Yes sure :)
Fixed in d2659dd2915c6c6ddbf600275d3371a0ef60cb8f